### ISSUE TITLE
[tools] install script for CentOS - multiple failures.

### ISSUE THREAD

**Issue by PapillonMcGill** [aces/Loris#4824]
- Need to change text to install composer on CentOS as installing as instructed in the script is not working.
    `sudo php composer-setup.php --install-dir=/usr/local/bin --filename=composer`  will work.

- chgrp for ../project dir, sudo not working (not permitted as on Ubuntu) (lines 169,183)

- directory modules/document_repository/user_uploads and modules/data_release/user_uploads are non-existant (lines 156-166).  Need to be created.  chown will fail on CentOS (see previous)

- can't create apache config for Loris (write permission missing)

**Comment by PapillonMcGill** [comment]
Sent a PR to solve message on installing composer https://github.com/aces/Loris/pull/4828
Waiting for future discussion on how to fix permissions

**Comment by cmadjar** [comment]
@PapillonMcGill Since we discussed this during the meeting, would you mind updating the ticket and maybe creating a PR with the outcome of the discussion. If I am not mistaken, it was:
- do a sudo test at the beginning of the script and exit if user has no sudo permission
- document what needs to be done in order to be able to install LORIS on CentOS.

Thanks!

**Comment by PapillonMcGill** [comment]
@cmadjar Checking for sudo permission would require to actually try to install (every instructions requiring privileges permission). 
@driusan is to discuss with Pam.
Documentation exist in the wiki. Suggest moving in the docs section after 21.0

**Comment by cmadjar** [comment]
@driusan did you talk to Pam about this? Is there anything to be done for this ticket or can it be closed in the end?

**Comment by johnsaigle** [comment]
@maltheism @PapillonMcGill Is this still a problem after the recent updates to the install script and CentOS documentation?

**Comment by PapillonMcGill** [comment]
@johnsaigle Yes, the install script still try to use sudo on RedHat based distro. Even in 22.0-release. Although the READMECentOS7.md file describe the step that need to be done and that the user doing it (lorisadmin) should have `sudo` permission, by default the install.sh script will fail. It is even not mentionned which sudo permission the user need. 
Remember by default, no single user have sudo permission, and each of these permissions (mdkir, chown, install composer, ...) have to be granted one by one.

I do believe that for RedHat based distro, the install.sh script should be run by a user with `root` access and that the script adjust the files ownership and permission accordingly after creation.

I need to confirm but I suspect that starting with RedHat 8, it is possible to designate a user to have full sudo acces during (OS) install. But it is not an obligation and less secure IMHO.

**Comment by johnsaigle** [comment]
@PapillonMcGill @racostas Was this an issue during CentOS testing for the 23.0 release?

**Comment by laemtl** [comment]
I noticed one issue with sudo and the install script during my manual tests, and some inconsistency in the install README.
